src/main/kotlin/app/revanced/patches/tiktok/misc/simspoof/patch/SimSpoofPatch.kt

```diff
@@ -0,0 +1,103 @@
+package app.revanced.patches.tiktok.misc.simspoof.patch
+
+import app.revanced.patcher.annotation.Description
+import app.revanced.patcher.annotation.Name
+import app.revanced.patcher.annotation.Version
+import app.revanced.patcher.data.BytecodeContext
+import app.revanced.patcher.extensions.addInstruction
+import app.revanced.patcher.extensions.addInstructions
+import app.revanced.patcher.extensions.instruction
+import app.revanced.patcher.extensions.softCompareTo
+import app.revanced.patcher.patch.BytecodePatch
+import app.revanced.patcher.patch.PatchResult
+import app.revanced.patcher.patch.PatchResultSuccess
+import app.revanced.patcher.patch.annotations.DependsOn
+import app.revanced.patcher.patch.annotations.Patch
+import app.revanced.patcher.util.proxy.mutableTypes.MutableClass
+import app.revanced.patches.tiktok.misc.integrations.patch.TikTokIntegrationsPatch
+import app.revanced.patches.tiktok.misc.settings.fingerprints.SettingsStatusLoadFingerprint
+import app.revanced.patches.tiktok.misc.settings.patch.TikTokSettingsPatch
+import app.revanced.patches.tiktok.misc.simspoof.annotations.SimSpoofCompatibility
+import org.jf.dexlib2.Opcode
+import org.jf.dexlib2.iface.ClassDef
+import org.jf.dexlib2.iface.Method
+import org.jf.dexlib2.iface.instruction.OneRegisterInstruction
+import org.jf.dexlib2.iface.instruction.formats.Instruction35c
+import org.jf.dexlib2.iface.reference.MethodReference
+import org.jf.dexlib2.immutable.reference.ImmutableMethodReference
+
+@Patch(false)
```

Comment:
Is this intended?

Reply:
Yes, because I think this patch targets some regions only. It should not be included by default. It also modifies the content of the tiktok delivered to the user based on the region it pretends to be in. 